eng/common/pipelines/templates/steps/docs-metadata-release.yml

```diff
@@ -34,12 +33,12 @@ steps:
   
 - pwsh: |
     git clone https://github.com/${{ parameters.TargetDocRepoOwner }}/${{ parameters.TargetDocRepoName }} ${{ parameters.WorkingDirectory }}/repo
-
+ 
     try {
       Push-Location ${{ parameters.WorkingDirectory }}/repo
-
-      Write-Host "git checkout ${{ parameters.SourceBranchName }}"
-      git checkout ${{ parameters.SourceBranchName }}
+      $setDefaultBranch = (git remote show origin | Out-String) -replace "(?ms).*HEAD branch: (\w+).*", '$1'
```

Comment:
Why not use the common template to get the default branch?

Reply:
Yes, will do